pkg/mapper/configmap/configmap.go

```diff
@@ -25,9 +26,10 @@ import (
 )
 
 type MapStore struct {
-	mutex sync.RWMutex
-	users map[string]config.UserMapping
-	roles map[string]config.RoleMapping
+	mutex           sync.RWMutex
+	users           map[string]config.UserMapping
+	roles           map[string]config.RoleMapping
+	arnLikeMappings []config.ARNLikeMapping
```

Comment:
To be consistent with the field naming for the other maps (`users` and `roles`), consider naming this field `arnLikes`